Source/Evolve/WarpXEvolveES.cpp

```diff
@@ -218,104 +218,3 @@ void WarpX::getLevelMasks(Vector<std::unique_ptr<FabArray<BaseFab<int> > > >& ma
         }
     }
 }
-
-
```

Comment:
The functions below are now defined (in modernized way) in `InitSpaceChargeFields.cpp`